modules/table/src/main/java/org/apache/ignite/internal/table/event/TableEventParameters.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.event;
+
+import java.util.UUID;
+import org.apache.ignite.internal.manager.EventParameters;
+import org.apache.ignite.internal.table.InternalTable;
+import org.apache.ignite.internal.table.TableSchemaView;
+
+/**
+ * Table event parameters.
+ * There are properties which associate with a concrete table.
+ */
+public class TableEventParameters implements EventParameters {
+    /** Table identifier. */
+    private UUID tableId;
+
+    /** Table name. */
+    private String tableName;
+
+    /** Table schema view. */
+    private TableSchemaView tableSchemaView;
+
+    /** Internal table. */
+    private InternalTable internalTable;
+
```

Comment:
What about affinity assignments?

Reply:
There is not an assignment field.
Is it definitely needed?